bullet_train-themes-tailwind_css/app/views/themes/tailwind_css/fields/_buttons.html.erb

```diff
@@ -1,23 +1,31 @@
 <%
 stimulus_controller = 'fields--button-toggle'
 form ||= current_fields_form
+
+# Initialize options.
 html_options ||= {}
-html_options[:id] ||= form.field_id(method)
-multiple ||= false
+options ||= {}
 other_options ||= {}
-options ||= options_for(form, method)
+button_field_options ||= options_for(form, method)
+
+# For backwards compatibility.
+options.merge!(html_options)if options.is_a?(Hash)
```

Comment:
Needs a space in here:

```suggestion
options.merge!(html_options) if options.is_a?(Hash)
```

Reply:
Done! https://github.com/bullet-train-co/bullet_train-core/pull/587/commits/c533c206944ed1acffd5d7ed7ad25da79dab1914